flink/v1.15/flink/src/main/java/org/apache/iceberg/flink/source/StreamingMonitorFunction.java

```diff
@@ -134,11 +135,17 @@ public void snapshotState(FunctionSnapshotContext context) throws Exception {
   public void run(SourceContext<FlinkInputSplit> ctx) throws Exception {
     this.sourceContext = ctx;
     while (isRunning) {
+      LOG.info("Start polling snapshots from snapshot id: {}, monitor snapshot number {}", lastSnapshotId,
```

Comment:
Use `debug` to prevent too many logs?

Reply:
Yeah this could get very excessive very quickly.

This should be a debug log as users can opt in via their logging configuration if need be.